#definition of the object classification output
#the id of the object
uint64 id
#the identified classes
ClassificationOutput[] classifications
